.github/workflows/build.yml

```diff
@@ -1,31 +1,46 @@
 name: build
 on:
   pull_request: {}
-  workflow_dispatch: {}
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
+    concurrency: ci-${{github.job}}-${{ github.ref }}
```

Comment:
what does `ci-...` mean for concurrency?

Reply:
It's the concurrency key. `ci` is just a static string. I gave it a better name. Only 1 task with a given concurrency key can run at a time. This prevents two test runs in the same branch from clobbering each other.